core/util/cursor.js

```diff
@@ -213,3 +215,23 @@ export default class Cursor {
         this._canvas.style.top = this._position.y + "px";
     }
 }
+
+Cursor.none = {
+    rgbaPixels: [],
+    w: 0,
+    h: 0,
+    x: 0,
+    y: 0,
+};
+
+Cursor.dot = {
+    rgbaPixels: Base64.decode(
+        "/////wAAAP//////" +
+        "AAAA/wAAAAAAAAD/" +
+        "/////wAAAP//////"
+    ),
+    w: 3,
+    h: 3,
+    x: 1,
+    y: 1,
+};
```

Comment:
These should be in `rfb.js` IMO as they are not a fundamental type for the `Cursor` class.

Reply:
OK